core/modules/user/user.module

```diff
@@ -1752,6 +1792,13 @@ function user_login($form, &$form_state) {
     '#title' => t('Password'),
     '#required' => TRUE,
     '#password_toggle' => TRUE,
+    '#attributes' => array(
+      // Add attributes to field to prevent spell-jacking and unwanted
+      // automatic changes caused by the browser.
+      'autocapitalize' => 'none',
+      'autocorrect' => 'off',
+      'spellcheck' => 'false',
```

Comment:
This block simply adds the desired attributes to both name and password for this specific high-level form.